### ISSUE TITLE
Subscriber custom fields

### ISSUE THREAD

**Issue by fulldecent** [mettle/sendportal#322]
Hello, this software looks great and glad to see this shared by the Mettle team.

Can we please consider to add per-subscriber profile fields?

1. Arbitrary fields can be created, any name, maybe some data type restrictions
2. These will apply to the subscriber list (and maybe inside a "workspace" not sure how workspaces work)
3. Can set each field for each subscriber
4. Templates can use any field using the same {{FIELD}} syntax

And then the obvious feature:

1. Templates support {{if}} {{else}} syntax to selectively include content. Or templates supports full handlebars/mustache syntax for template language.

**Comment by carallel** [comment]
Agreed - this would be great!

**Comment by jdevalk** [comment]
Yes. I can't really use this for most of my use cases until it has this...

**Comment by fulldecent** [comment]
Me too. I made a dead simple merge mailer that works with Mailgun.

Hosted from a web page. No server. Can fork and run locally. We're using it in production.

https://github.com/fulldecent/sandy-blaster
